### PR TITLE
RD-6525 Move `SERVER_HOST` and `SERVER_PORT` constants to JSON configuration file

### PR DESCRIPTION
**Pull request by qooban**
## Description
This PR changes `SERVER_HOST` and `SERVER_PORT` constants into configurable parameters in `conf/app.json` file. This should allow others to use Stage package in docker container.

Direct trigger for introducing such change: RD-6342.

## Screenshots / Videos
N/A

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
Local environment (`devServer`) works fine.

System tests:
1. [Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4025/) - failed on running backend (no configuration available as `app.json` is handled by Cloudify Manager, fix: https://github.com/cloudify-cosmo/cloudify-stage/pull/2305/commits/34212e552729b015f36df618908784d635cb4c93)
2. [Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4044/) - backend started successfully, tests seems passing, so I feel comfortable to merge it

## Documentation
Updated `conf/README.md` file.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "noEmit": true,
         "esModuleInterop": true,
+        "resolveJsonModule": true,
```
Comment (kubama) [inline tsconfig.json:5]: Why do we need this change?

Comment (qooban) [reply tsconfig.json:5]: `backend/config.ts` bases on resolving JSON modules and right now `devServer.ts` requires it.
I think it's not harmful, but if you have any better solution I'm open to change it.

Comment (kubama) [reply tsconfig.json:5]: But there is no change in `backend/config.ts`... so it worked prior to this PR without this change?

Comment (qooban) [reply tsconfig.json:5]: > But there is no change in `backend/config.ts`... so it worked prior to this PR without this change?

Yes (there is no change in that file) and yes (it worked as it was not imported by `devServer.ts`).
I assume that TS uses `tsconfig.json` for type checking in main folder.

As I said if you have better solution I'm open to change it.

Comment (kubama) [reply tsconfig.json:5]: > it worked as it was not imported by `devServer.ts`

Ah, it makes sense now :+1:

**Review by kubama — DISMISSED** [review]

**Review by tehasdf — APPROVED** [review]
lovely

**Review by tehasdf — APPROVED** [review]

**Review by kubama — APPROVED** [review]

**Pull request merged**